src/camera.py

```diff
@@ -0,0 +1,76 @@
+from os import getcwd
+
+from picar.SunFounder_PCA9685 import Servo
+from picar import filedb
+
+
+'''
+Camera class.
+
+Uses SunFounder Servo class to control camera servos allowing for pan and tilt.
+Uses SunFounder filedb to configure offset for pan and tilt functions.
+'''
+class Camera:
+    PAN_CHANNEL = 1
+    TILT_CHANNEL = 2
+
+    def __init__(self, bus_number=1):
+        self._DB = filedb.fileDB(getcwd() + "/../SunFounder_PiCar-V/remote_control/remote_control/driver/config")
+        self._pan_offset = int(self._DB.get('pan_offset', default_value=0))
+        self._tilt_offset = int(self._DB.get('tilt_offset', default_value=0))
+        self._pan_servo = Servo.Servo(self.PAN_CHANNEL, bus_number=bus_number, offset=_self.pan_offset)
```

Comment:
Can you try to limit these lines to around 80 characters? Break them up at reasonable points like the commas or something.